Mathlib/CategoryTheory/ChosenFiniteProducts.lean

```diff
@@ -484,4 +485,54 @@ end ChosenFiniteProductsComparison
 
 end ChosenFiniteProducts
 
+open Limits MonoidalCategory ChosenFiniteProducts
+
+variable {C : Type u} [Category.{v} C] [ChosenFiniteProducts C]
+  {D : Type u₁} [Category.{v₁} D] (F : C ⥤ D)
+  [ChosenFiniteProducts D]
```

Comment:
In a few situations, it is desirable to have instance parameters at the beginning rather than at the end:
```suggestion
  {D : Type u₁} [Category.{v₁} D] [ChosenFiniteProducts D] (F : C ⥤ D)
```